autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -133,11 +139,20 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             Default value is 0.2 if `hyperparameter_tune = True`, otherwise 0.1 is used by default. 
         num_bagging_folds : (int)
             Number of folds used for bagging of models. When `num_bagging_folds = k`, training time is roughly increased by a factor of `k` (set = 0 to disable bagging). 
-            Disabled by default, but we recommend values between 5-10 to maximize predictive performance. 
+            Disabled by default, but we recommend values between 5-10 to maximize predictive performance.
+        num_bagging_sets : (int)
+            Number of repeats of kfold bagging to perform. Total number of models trained during bagging = num_bagging_folds * num_bagging_sets.
```

Comment:
Can we add one more documentation sentence describing why `num_bagging_sets` > 1 is useful vs say just setting `num_bagging_folds` to a large number like 100?

Reply:
Added